pom.xml

```diff
@@ -23,16 +23,43 @@ under the License.
 
   <groupId>com.getindata</groupId>
   <artifactId>flink-http-connector</artifactId>
-  <version>1.0-SNAPSHOT</version>
+  <version>0.1-SNAPSHOT</version>
```

Comment:
please use X.X.X pattern for versioning. 

So in our case we will have 0.1.0-SNAPSHOT